lib/server.js

```diff
@@ -27,8 +27,16 @@ function registerGet(parsedEndpoint) {
 }
 
 module.exports = function(callback) {
+  var PORT = callback.port || 5555;
+
   if (typeof callback !== 'function') callback = function() {}
 
+  if(typeof PORT != "number"){
```

Comment:
Won't this will always be true since the command-line arg is a string? Instead, we can do
```
  if (isNaN(PORT)) {
      console.log('port should be a valid number')
      callback({message: 'invalid port'})
      return
  }
```

Also, let's maintain 2 spaces for indentation :)